### DISCUSSION TITLE
MavSysID and MavConpID

### DISCUSSION THREAD

**Discussion by InanixFR** [svpcom/wfb-ng#213] [Ideas]
Hi all,

For the moment, WFB send a RSSI status Mavlink message with a static Vehicule Sys ID and a static component id hard coded line 65 in this file https://github.com/svpcom/wifibroadcast/blob/b610c5030019f41e9b04ea967abfbf6a39cbd709/telemetry/proxy.py

Following the mavlink documentation (https://mavlink.io/en/messages/common.html#MAV_COMP_ID_TELEMETRY_RADIO) I think the component ID 68 should be a better choice.

By the way, it would be great if we can configure this values (compid AND sysid) in the configuration file!

**Comment by svpcom** [comment]
Done in #214
